src/Hazelcast.Net.Tests/Serialization/Compact/CompactReaderWriterExtensionsTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#nullable enable
+
+using System;
+using Hazelcast.Core;
+using Hazelcast.Serialization;
+using Hazelcast.Serialization.Compact;
+using NUnit.Framework;
+
+namespace Hazelcast.Tests.Serialization.Compact;
+
+[TestFixture]
+public class CompactReaderWriterExtensionsTests
+{
+    [Test]
+    public void WriteAny()
```

Comment:
Just a comment. What about moving the last line of this test to ReadAny() and removing this test altogether? ReadAny seems to be testing WriteAny as well.

Reply:
makes sense, doing